### ISSUE TITLE
Use NLP to parse teacher goal setting language

### ISSUE THREAD

**Issue by neilthawani** [neilthawani/classinsight-prototype#206]
From Amy - Neil, could you do Bag of Words parsing for teacher goal setting? A sentence that’s meaningful enough so that we could do something with it later? Can we test it using interview data gathered in November?

**Comment by neilthawani** [comment]
See email from Andi with subject - Reflection Session Transcripts and Artifacts

**Comment by neilthawani** [comment]
tl;dr: Teachers want students to engage in certain codes more, more student talk, less teacher talk; what do teachers think about 

From Sherice: We want goal setting to occur at all the grain sizes. Currently goal setting only happens at the coarse grained/TalkRatio level.

Teachers notice the patterns of discourse. How do they set goals around those patterns?
And with regard to the words they're saying (Transcript visualization) - how do you elicit types of talk?

Should there be 3 different goal setting visualizations? Should there be one?
How do we go beyond "I want to increase this type of talk more" or "I want to increase student talk percentage to x%." ?
